source/agora/cli/checkvtable/types.d

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+
+    Contains types to check the offset of the vtable.
+
+    Copyright:
+        Copyright (c) 2020 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.cli.checkvtable.types;
+
+public import scpd.scp.SCPDriver;
+
+import std.meta;
+
+extern (C++) public class TestA
```

Comment:
We don't need this class anymore right? It was only used during development. We only care about the SCPDriver class.

Reply:
> We don't need this class anymore right? It was only used during development. We only care about the SCPDriver class.

I needed a function that could handle several things using 'foreach statement'. I needed more than one class to test this.

Reply:
I removed TestA.